clients/csharp-dotnet-core/voice-assistant-test/tool/DialogResult.cs

```diff
@@ -177,10 +177,27 @@ public static int CompareJObjects(JObject expected, JObject actual, bool enableL
                         string actualResult = actualValue.ToString();
                         string expectedResult = expectedValue.ToString();

```

Comment:
expectedResult can now be very long if it has the || operator. Too long to fully trace below? I'm wondering if it's too long to print as is for traces. We can deal with it in another PR if needed.